aptoide-database/src/main/java/cm/aptoide/pt/database/accessors/StoreAccessor.java

```diff
@@ -40,8 +36,4 @@ public void save(Store store) {
   public Observable<List<Store>> getAsList(long storeId) {
     return database.getAsList(Store.class, Store.STORE_ID, storeId);
   }
-
-  public Observable<List<Store>> getAsList(String storeName) {
-    return database.getAsList(Store.class, Store.STORE_NAME, storeName);
-  }
 }
```

Comment:
more methods here can probably be deleted as well.

Reply:
Not deleting these ones because these are realm classes that will be deleted later on, no point on deleting it now